pkgs/applications/video/gpu-screen-recorder/gpu-screen-recorder-gtk.nix

```diff
@@ -1,34 +1,37 @@
-{ stdenv
-, lib
-, fetchurl
-, pkg-config
-, desktop-file-utils
-, makeWrapper
-, meson
-, ninja
-, gtk3
-, libayatana-appindicator
-, libpulseaudio
-, libdrm
-, gpu-screen-recorder
-, libglvnd
-, libX11
-, libXrandr
-, wayland
-, wrapGAppsHook3
-, wrapperDir ? "/run/wrappers/bin"
+{
+  stdenv,
+  lib,
+  fetchurl,
+  pkg-config,
+  addDriverRunpath,
+  desktop-file-utils,
+  makeWrapper,
+  meson,
+  ninja,
+  gtk3,
+  libayatana-appindicator,
+  libpulseaudio,
+  libdrm,
+  gpu-screen-recorder,
+  libglvnd,
+  libX11,
+  libXrandr,
+  wayland,
+  wrapGAppsHook3,
+  wrapperDir ? "/run/wrappers/bin",
 }:
 
 stdenv.mkDerivation {
   pname = "gpu-screen-recorder-gtk";
-  version = "unstable-2024-07-05";
+  version = "unstable-2024-09-02";
 
   # Snapshot tarballs use the following versioning format:
   # printf "r%s.%s\n" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
   src = fetchurl {
-    url = "https://dec05eba.com/snapshot/gpu-screen-recorder-gtk.git.r311.c611c51.tar.gz";
-    hash = "sha256-86EdmeZ2dlffSfJOrTVGPtYyL3j6DmCQIALX2rpeS1Y=";
+    url = "https://dec05eba.com/snapshot/gpu-screen-recorder-gtk.git.r374.4f267e6.tar.gz";
+    hash = "sha256-6LKrNb3ncfQNU23H47V/IlJi6xbynoFV8VKHfTo/mmc=";
```

Comment:
Not sure how to grab the URL and hash based on stable tag.

Reply:
The tagged releases can now be accessed as tar.gz files by replacing the r374.4f267e6 part with the tag name. For example https://dec05eba.com/snapshot/gpu-screen-recorder-gtk.git.4.1.11.tar.gz